composer.json

```diff
@@ -11,7 +11,7 @@
     },
 
     "require-dev": {
-        "phpunit/phpunit": "4.8.*",
+        "phpunit/phpunit": "^6.5",
         "phing/phing": "2.*",
```

Comment:
Doing the smallest possible PHPUnit upgrde that will allow us to solve our warnings and still have PHP 7.{0,1,2,3,4} support.